pyhap/accessory_driver.py

```diff
@@ -510,8 +529,10 @@ def stop(self):
         self.accessory_thread.join()
 
         logger.debug("Stopping mDNS advertising")
-        self.advertiser.unregister_service(self.mdns_service_info)
-        self.advertiser.close()
+        try:
+            self.advertiser.close()
+        except:
+            pass
```

Comment:
Can we log the exception here? it will help in debugging

Reply:
Would a debug statement be enough, since it isn't relevant for any implementation?

Reply:
Sounds good